src/components/newdapp/index.js

```diff
@@ -93,6 +101,59 @@ export default class DevkitNewDapp extends Component {
         }
     };
 
+    import = (evt) => {
+        evt.preventDefault();
+        var project=this.state.projectName;
+        if(project=="") {
+            alert("Please give the project a name.");
+            return;
+        }
+        if(!project.match(/^([a-zA-Z0-9-]+)$/)) {
+            alert('Illegal projectname. Only A-Za-z0-9 and dash (-) allowed.');
+            return;
+        }
+        var contentJSON="";
+        const ok=(e)=>{
+            e.preventDefault();
+            if(!contentJSON) {
+                alert("No JSON provided. Paste it into the text area and try again.");
+                return;
+            }
+            var dappfileJSONObj;
+            try {
+                dappfileJSONObj=JSON.parse(contentJSON);
+            }
+            catch(e) {
+                alert("Could not parse JSON, is it properly pasted?");
```

Comment:
Properly pasting is not a problem, per se. The problem is simply an invalid JSON in this case. 
Other suggestions for the error message: "Invalid JSON data.", "Could not load JSON", or, more generally, "Invalid project data.", "Invalid project description", "Could not understand project description".

Reply:
I was thinking the only user available option to fix the problem would be an error in copy/paste. That's why the error message suggests that. I don't have a hard stance on it though.